tests/IntegrationTests/Pingable.slice

```diff
@@ -4,6 +4,6 @@ mode = Slice1
 
 module IceRpc::IntegrationTests
 
-interface Pingable {
+interface Pingable : Ice::Object {
```

Comment:
I don't understand this update, and I'd rather not do that.

Reply:
The previous version of `PingableService` was doing

```
private class PingableService : Service, IPingableService, IIceObjectService
```

My understanding is we need `Ice::Object` inheritance to test `Ice_operations`, Maybe we should use a separate service for the `Ice_operation` test?